tools/run_pr_builder.sh

```diff
@@ -9,7 +9,6 @@
 set -eux
 
 export GOPATH=${WORKSPACE}
-export PATH=${PATH}:/usr/local/go/bin:/var/lib/jenkins/node-v5.11.1-linux-x64/bin
```

Comment:
Is it no longer necessary for this script to add node to the path ? Is it handled upstream by the calling process ?  You might have mentioned this in IRL, just want to dbl check.